frontend/forms/OBSBasicSettings.ui

```diff
@@ -8784,12 +7497,6 @@
   <tabstop>simpleOutPreset</tabstop>
   <tabstop>simpleOutAdvanced</tabstop>
   <tabstop>simpleOutCustom</tabstop>
-  <tabstop>simpleOutRecTrack1</tabstop>
```

Comment:
Maybe I missed it in the implementation, but is the functionality to use tab to cycle through these controls retained?

Reply:
I've spent a few days working on this. It works fine for checkboxes, but radio buttons behaves weird. I just can't make them tab correctly. I think I'll just drop my work in few days and maybe somebody more experienced than me will be able to understand what is happening.

Reply:
You can use setTabOrder when these widgets are being created

https://doc.qt.io/qt-6/qwidget.html#setTabOrder